psycop/common/model_training_v2/trainer/data/test_dataloaders.py

```diff
@@ -33,3 +38,18 @@ def test_vertical_concatenator(tmpdir: Path):
         ParquetVerticalConcatenator(
             paths=[str(p) for p in parquet_paths] + ["non_existent_path"],
         ).load()
+
+
+def test_dataloader_filterer():
```

Comment:
On a first look, this test checks types and the logic in line 48. Perhaps we don't need it then? :-)

The interesting logic is in:
* psycop/common/model_training_v2/trainer/data/data_filters/geography.py
* psycop/common/data_inspection/visits_by_hospital_units/make_geographical_split.py

I would love some tests of those ❤️